### DISCUSSION TITLE
How would I best go about creating a custom dashboard/views beyond the pre-built support?

### DISCUSSION THREAD

**Discussion by axolmain** [datalust/seq-tickets#2049] [Help] [answered]
Hello,

I'm interested in essentially creating a wrapper over the current server GUI. I want to create some custom pages to replace how the current dashboard feature so I can add more features (e.g. interactive tables and more custom graphs). Is there a way to directly change the pre-built site's views/features?

If not, what would be the best way to create my own web app for showing data from the logs I store in Seq? I wasn't able to see any clear documentation or past examples for this use case.

Thanks!

**Comment by liammclennan** [comment, marked as answer]
Hi @axolmain ,

You can use the HTTP API to read data from Seq, or you can use tools like Tableau or PowerBI.

**Reply by axolmain** [reply]
Gotcha. Thanks @liammclennan !

**Reply by axolmain** [reply]
Do you know if it's possible to edit the UI html/js directly?

**Reply by liammclennan** [reply]
You can't modify Seq. You can build your own application that accesses Seq data via the API.
